util/logging.c

```diff
@@ -91,7 +97,12 @@ int nvme_submit_passthru(int fd, unsigned long ioctl_cmd,
 {
 	struct timeval start;
 	struct timeval end;
-	int err;
+	int err = 0;
```

Comment:
This is an unrelated change, not really needed.

Reply:
Yes so fixed.